tmpl/README.md.j2

```diff
@@ -83,7 +81,7 @@ machines, which is only possible with a Jenkins pipeline (GitHub actions is runn
 ### Pre-commit on GitHub actions
 
 `.github/workflows/pre-commit.yml` contains a hook that will trigger the creation of your environment (unpinned) on the GitHub actions server and
-then run pytest as well as various formatters and linters through pre-commit. This hook is only triggered upon pushes to the main branch (in general: don't do that)
+then run various formatters and linters through pre-commit. This hook is only triggered upon pushes to the main branch (in general: don't do that)
```

Comment:
Does it not run pytest (which is not in pre-commit)? (I admittedly didn't double-check.)

Reply:
I think it currently doesn't (or I cannot find it).